imageprocessor/imageprocessor-core/src/main/java/pl/edu/uj/kimage/processingFlow/ImageProcessingFlowFactory.java

```diff
@@ -7,6 +7,9 @@
 import pl.edu.uj.kimage.plugin.FlowStep;
```

Comment:
why this package is `processingFlow`? Is inconsistent with the conventions - should be `processing.flow`